templates/definition/charger/teslalogger.yaml

```diff
@@ -0,0 +1,55 @@
+template: teslalogger
+products:
+  - description:
+      generic: TeslaLogger
+requirements:
+  description:
+    en: Open source Tesla data logger https://github.com/bassmaster187/TeslaLogger, works only with "Teslalogger vehicle"
+    de: Open Source Tesla Datenlogger https://github.com/bassmaster187/TeslaLogger, funktioniert nur in Verbdinung mit "TeslaLogger Fahrzeug"
+params:
+  - name: url
+    required: true
+    example: http://192.0.2.2
+  - name: port
+    example: 5000
+    default: 5000
+  - name: id
+    description:
+      de: TeslaLogger CarID
+      en: TeslaLogger CarID
+    default: 1
+render: |
+  type: custom
+  status:
+    source: combined
+    charging:
+      source: http
+      uri: {{ .url }}:{{ .port }}/currentjson/{{ .id }}
+      jq: .charging
+    plugged:
+      source: go
```

Comment:
Geile Lösung- aber das kannst Du sicher auch einfach in JQ und damit mit einem HTTP Request machen.